<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <attribute name="title">
         <string>Main</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>791</width>
           <height>511</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_10">
            <property name="text">
             <string>Edit Selected Virtual Machine</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_9">
            <property name="text">
             <string>Start Selected Virtual Machine</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButton_11">
            <property name="text">
             <string>Delete Selected Virtual Machine</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>New Virtual Machine</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5"/>
          </item>
          <item row="0" column="1">
           <widget class="QListView" name="listView"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <attribute name="title">
         <string>Settings</string>
        </attribute>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>791</width>
           <height>521</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>QEMU</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>761</width>
             <height>481</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="6" column="3">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>qemu-img Path</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QLineEdit" name="lineEdit_6"/>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>qemu-system-i386 path</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>qemu-system-mips64el path</string>
              </property>
             </widget>
            </item>
            <item row="7" column="3">
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>qemu-system-aarch64 path</string>
              </property>
             </widget>
            </item>
            <item row="9" column="3">
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEdit_5"/>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>qemu-system-x86_64 path</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>qemu-system-ppc path</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>qemu-system-arm path</string>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QLineEdit" name="lineEdit_7"/>
            </item>
            <item row="8" column="3">
             <widget class="QPushButton" name="pushButton_12">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Update</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget_5">
           <property name="geometry">
            <rect>
             <x>1</x>
             <y>0</y>
             <width>781</width>
             <height>501</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_13">
              <property name="text">
               <string>Check for updates</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>GitHub</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Codeberg</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_2">
              <item>
               <property name="text">
                <string>Everytime I run this program</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Every day</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Every week</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Every month</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Never</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Update notify frequency</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_14">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Update mirror</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Update channel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBox_3">
              <item>
               <property name="text">
                <string>stable</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>pre-release</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>About</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>781</width>
             <height>481</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Built on Python and PyQt technology, licensed under GNU General Public License 3.0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>WARNING: This program comes with ABSOLUTELY NO WARRANTY under applicable law. Please see the GNU GPL license for details.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="text">
               <string>EmuGUI</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>781</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>EmuGUI v0.0.1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
